zerver/migrations/0060_move_avatars_to_be_uid_based.py

```diff
@@ -28,7 +28,7 @@ def move_local_file(type: Text, path_src: Text, path_dst: Text) -> None:
     if not os.path.exists(src_file_path):
         # This is likely caused by a user having previously changed their email
         raise MissingAvatarException()
-        return
+        return  # type: ignore # ->warn-unreachable
```

Comment:
This I think might be a bug, where raising an exception that just passes, followed by a `return` statement, flags said `return` statement as unreachable.

Reply:
No, there’s no such thing as an “exception that just passes”—anything following a `raise` statement _is_ unreachable.

Reply:
Agreed, was about to make the same comment.

Reply:
Okay understood, I only ask this because the `MissingAvatarException` class contains the one line, `pass`, but I see I misinterpreted that.